README.md

```diff
@@ -226,27 +226,60 @@ The VC++ linker supports `/SOURCELINK` [switch](https://docs.microsoft.com/en-us
 
 ## PDB distributions
 
+There are three main ways to distribute PDBs: via a dedicated .snupkg, including them in the main package, and embedding them directly into the assembly. Each has advantages and drawbacks, depending on your use case.
+
+|                                         | snupkg                      | Include in main package | Embed in assembly |
+|-----------------------------------------|-----------------------------|-------------------------|-------------------|
+| No user opt-in required to load symbols | ✅                          | ❌                      | ✅                |
+| No increase in size of main package     | ✅                          | ❌                      | ❌                |
+| No increase in size of assemblies       | ✅                          | ✅                      | ❌                |
+| Supported by all package feeds          | ❌ (Supported on NuGet.org) | ✅                      | ✅                |
+| Supports all symbol types               | ❌ (Portable symbols only)  | ✅                      | ✅                |
```

Comment:
Only Portable PDB can be embedded in assembly. 

Not sure I would even include this line - Windows PDB are legacy for managed code. They should only be used for C++.

Reply:
Removed the line and added a word to note that this section is for managed code.

Reply:
I believe I've addressed your concerns, so I'm marking this thread as resolved. If you disagree or have any other issues, please reactivate and provide updated details. Thanks!